velox/common/caching/CachedFactory.h

```diff
@@ -184,7 +97,7 @@ class CachedFactory {
   // Clear the cache and return the current cache status
   SimpleLRUCacheStats clearCache() {
     std::lock_guard l(cacheMu_);
-    cache_->free(cache_->maxSize());
+    cache_->clear();
     return cache_->getStats();
```

Comment:
I think this might not be semantically correct. Clear() should also clear all stats as well. Maybe add stats clearing logic in clear() and here we can make a copy of the stats before clear() then return the copy after clear.